Public/robots.txt

```diff
@@ -1,25 +1,4 @@
 Sitemap: https://swiftpackageindex.com/sitemap.xml
 
-User-agent: Bytespider
-Disallow: /
-
-User-agent: GPTBot
-Disallow: /
-
-User-agent: ClaudeBot
-Disallow: /
-
-User-agent: anthropic-ai
-Disallow: /
-
-User-agent: Applebot
-Disallow: /
-
-User-agent: Google-Extended
-Disallow: /
-
-User-agent: Cohere-ai
-Disallow: /
-
-User-agent: Seekr
-Disallow: /
```

Comment:
Mmm hold on, we _don't_ want the AI bots crawling the site, do we?

Reply:
I wrote this in the comment:

> We were no longer using the AI bot rules so I removed them to keep it clear that we are not dealing with that in robots.

These rules are not what is blocking the bots, and this is not a comprehensive list, so it's misleading for us to have these rules here. They are not being used.

Reply:
Maybe we should indicate that, but I think it's also a good place to indicate _intent_. `robots.txt` is the only place where you can communicate you objection to AI crawling quite explicitly and if it doesn't do any harm I'd be in favour of doing so.